packages/ui-reactive/src/Unlocking.tsx

```diff
@@ -52,24 +70,28 @@ export class UnlockingDisplay extends React.PureComponent<Props> {
     const { className, controllerId, style, t, unlockings } = this.props;
 
     if (!unlockings || !controllerId) return null;
+
     const labels: {locked: string, remaining: string } = {
       locked: t('locked '),
       remaining: t(' blocks left')
     };
-    const locked = unlockings.filter((chunk) => this.remainingBlocks(chunk.era).gtn(0));
+    // select the Unlockchunks that can't be unlocked yet.
+    const filteredUnlockings = unlockings.filter((chunk) => this.remainingBlocks(chunk.era).gtn(0));
+    // group the Unlockchunks that have the same era and sum their value
+    const groupedUnlockings = filteredUnlockings.length ? this.groupByEra(filteredUnlockings) : undefined;
 
     return (
-      <div>
-        {locked.map((unlocking,index) => (
+      <>
+        { groupedUnlockings && [...groupedUnlockings].map(([eraString, value],index) => (
           <div
             className={className}
             style={style}
             key={index}
           >
-            {labels.locked}{formatBalance(unlocking.value)} ({this.remainingBlocks(unlocking.era).toString()}{labels.remaining})
+            {labels.locked}{formatBalance(value)} ({this.remainingBlocks(new BlockNumber(eraString)).toString()}{labels.remaining})
```

Comment:
This is probably better suited via a,

```js
t('locked {{balance}} ({{remaining}} blocks left), {
  replace: {
    balance: formatBalance(value),
    remaining: this.remainingBlocks(new BlockNumber(eraString))
  }
});
```

i.e. use interpolation on the i18n string

Reply:
thanks it's much nicer that way indeed.